react-markdown/react-markdown.d.ts

```diff
@@ -0,0 +1,182 @@
+// Type definitions for react-markdown
+// Project: https://github.com/rexxars/react-markdown
+// Definitions by: Greg Rosenbaum <https://github.com/GregRos>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="../react/index.d.ts"/>
+declare namespace ReactMarkdown {
```

Comment:
Prefer to write typings in the module rather than in a namespace.
This means removing the `declare namespace ReactMarkdown` wrapper as well as the `declare module "react-markdown" { .. }` wrapper at the bottom of the file and just using the file itself as the module's scope.
This also means using `import * as React from "react";` rather than using the global `React` namespace.